test/functional/options/defaults_spec.lua

```diff
@@ -328,3 +331,277 @@ describe('XDG-based defaults', function()
     end)
   end)
 end)
+
+
+describe('stdpath()', function()
+  -- Returns an OS friendly version of the unix_path.
+  local function alter_path(unix_path)
+    if helpers.iswin() then
+      return string.gsub(unix_path, '/', '\\')
```

Comment:
`string.gsub` returns two items (note: not a list of two items, two items). Use `select` or first assign to a variable and then return it (better second because I remember some issues with select, though do not remember which exactly). This is why you get `2` on AppVeyor.